py3status/modules/screenshot.py

```diff
@@ -4,82 +4,195 @@
 
 Display a 'SHOT' button in your i3bar allowing you to take a screenshot and
 directly send (if wanted) the file to your online server.
-When the screenshot has been taken, 'SHOT' is replaced by the file_name.
+When the screenshot has been taken, the file_name is shown. Right click to clean the text
+by erasing all available placeholders variables when you don't need to see it anymore.
 
 By default, this modules uses the 'gnome-screenshot' program to take the screenshot,
 but this can be configured with the `screenshot_command` configuration parameter.
 
+The upload can be configured with the `post_processing` parameter. It accepts the
+following values:
+    * `"scp"`: upload by `scp` protocol, needs `upload_server`, `upload_user` and
+      `upload_path` to be configured.
+    * `"lutim"`: upload on a [Lutim](https://framagit.org/luc/lutim) instance, needs
+      `upload_server` to be configured (URL).
+    * `"custom:path_to_script"`: specify a custom script which will be called after the
+      screenshot has been taken and whose first argument will be the path to the
+      screenshot.
+    * `None`: No post processing. Same effect as parameter `push` set to `False`.
+
+The `push` parameter is kept for compatibility reasons.
+
 Configuration parameters:
-    cache_timeout: how often to update in seconds (default 5)
-    file_length: generated file_name length (default 4)
-    push: True/False if yo want to push your screenshot to your server
+    file_length: generated random file_name length (default 4)
+    format: displayed text format (default "{filename}|{status}")
+    image_format: format of the screenshot (default 'jpg')
+    logfile: path of the logfile, recommended if you use a image hoster, as you will keep
+        the picture URL and the deletion URL (default None)
+    open_webbrowser: when uploading to an image hoster, open the sent capture in the
+        webbrowser (default False)
+    post_processing: specify what to do with the screenshot (default 'scp')
```

Comment:
Maybe allow this to be a list so we can do multiple things - could the web browser fit in with this?